src/key_schedule.cpp

```diff
@@ -416,8 +401,8 @@ GroupKeySource::decrypt(const bytes& sender_data_secret,
   return MLSPlaintext{ ct.group_id,
                        ct.epoch,
                        { SenderType::member, sender_data.sender },
-                       ct.content_type,
                        ct.authenticated_data,
+                       ct.content_type,
```

Comment:
It might be good to move the construction logic from `MLSPlaintext::MLSPlaintext` to here.  That way only this file / struct would have to know about `MLSCiphertextContent`.  The content of that `MLSPlaintext` constructor seems like it would more properly go in an `operator>>` anyway.